common/common-ui/src/main/res/values/attrs-text-input.xml

```diff
@@ -19,6 +19,7 @@
         <attr name="android:hint" />
         <attr name="android:text" />
         <attr name="android:minLines" />
+        <attr name="capitalizeKeyboard" format="boolean" />
```

Comment:
nit: capitalize keyboard or text? the parameter for TextView is `textAllCaps`, we could reuse it if you wanted.

Reply:
The original implementation was capitalizeKeyboard, which is slightly different. In this case, the keyboard is displayed in all caps by default, but the user could still switch to lowercase (or use an external keyboard) and text wouldn't be capitalized automatically. I did it this way to mimic the original behavior as much as possible.